func.go

```diff
@@ -378,17 +378,21 @@ func funcFromEntries(v interface{}) interface{} {
 		case map[string]interface{}:
 			var (
 				key   string
+				keyOk bool
```

Comment:
You don't need this variable.

Reply:
Fixed, thought maybe want it to be more explicit